src/databases/mongodb/repository/mongodbManager.js

```diff
@@ -3,7 +3,8 @@ const { MongoClient } = require('mongodb')
 const crearConexion = async function(user, password, dbName) {
     return new Promise(async function(resolve, reject) {
         try {
-            const connStr = `mongodb+srv://${user}:${password}@cluster0.p7sle.mongodb.net/${dbName}?retryWrites=true&w=majority&useUnifiedTopology=true`;
+            const connStr = `mongodb://localhost:27017/?readPreference=primary&appname=MongoDB%20Compass%20Community&ssl=false`;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Backend/pullRequest?prid=7180713)
```suggestion
            const connStr = "mongodb://localhost:27017/?readPreference=primary&appname=MongoDB%20Compass%20Community&ssl=false";
```